UPGRADE-2.0.md

```diff
@@ -3,33 +3,4 @@
 ## Needed actions
 This is the list of actions that you need to take when upgrading this bundle from the 1.x to the 2.0 version:
 
- *  Since 1.x uses `nelmio/alice` <= 2 and 2.0 switched to 3 with `theofidry/alice-data-fixtures`: 
```

Comment:
same here .. I guess we need to leave this file as is and add a new UPGRADE-3.0.md instead